### PR TITLE
Apply suggestions from grammar check

### PR DESCRIPTION
**Pull request by Shute052**
Apply suggestions from grammar check in the comments

### CODE REVIEW

File: benches/input_map.rs [hunk 1]
```diff
@@ -75,7 +75,7 @@ pub fn criterion_benchmark(c: &mut Criterion) {
     });
     let mut which_pressed_group = c.benchmark_group("which_pressed");
 
-    // Constructing our test app / input stream outside of the timed benchmark
+    // Constructing our test app / input stream outside the timed benchmark
```
Comment (alice-i-cecile) [inline benches/input_map.rs:78]: Both are acceptable.

File: examples/arpg_indirection.rs [hunk 2]
```diff
@@ -23,7 +23,7 @@ fn main() {
             PreUpdate,
             copy_action_state.after(InputManagerSystem::ManualControl),
         )
-        // Try it out, using QWER / left click / right click!
+        // Try it out, using QWER / left-click / right-click!
```
Comment (alice-i-cecile) [inline examples/arpg_indirection.rs:26]: Definitely an improvement.

File: examples/axis_inputs.rs [hunk 3]
```diff
@@ -35,7 +35,7 @@ fn spawn_player(mut commands: Commands) {
                 .insert(Action::Move, DualAxis::left_stick())
                 // Let's bind the right gamepad trigger to the throttle action
                 .insert(Action::Throttle, GamepadButtonType::RightTrigger2)
-                // And we'll use the right stick's x axis as a rudder control
+                // And we'll use the right stick's x-axis as a rudder control
```
Comment (alice-i-cecile) [inline examples/axis_inputs.rs:38]: Clear mistake

File: examples/consuming_actions.rs [hunk 4]
```diff
@@ -21,7 +21,7 @@ fn main() {
         .add_systems(Update, report_menus)
         .add_systems(Update, open_main_menu)
         .add_systems(Update, open_sub_menu)
-        // We want to ensure that if both the main menu and submenu are open
+        // We want to ensure that if both the main menu and submenu are open,
```
Comment (alice-i-cecile) [inline examples/consuming_actions.rs:24]: Mild improvement.

File: examples/consuming_actions.rs [hunk 5]
```diff
@@ -67,8 +67,9 @@ fn open_sub_menu(action_state: Res<ActionState<MenuAction>>, mut menu_state: Res
     }
 }
 
-// We want to be sure that e.g. the submenu is closed in preference to the main menu if both are open
-// If you can, use a real focus system for this logic, but workarounds of this sort are necessary in bevy_egui
+// We want to ensure that, e.g., the submenu is closed in preference to the main menu if both are open.
+// If you can, use a real focus system for this logic.
```
Comment (alice-i-cecile) [inline examples/consuming_actions.rs:71]: Slightly clearer.

File: examples/consuming_actions.rs [hunk 6]
```diff
@@ -77,7 +78,7 @@ fn close_menu<M: Resource + Menu>(
     if action_state.pressed(&MenuAction::CloseWindow) && menu_status.is_open() {
         println!("Closing the top window, as requested.");
         menu_status.close();
-        // Because the action is consumed, further systems won't see this action as pressed
+        // Because the action is consumed, further systems won't see this action as pressed,
```
Comment (alice-i-cecile) [inline examples/consuming_actions.rs:81]: Mild imrpvement.

File: examples/mouse_wheel.rs [hunk 7]
```diff
@@ -53,9 +53,9 @@ fn zoom_camera(
     // Up and right axis movements are always positive by default
     let zoom_delta = action_state.value(&CameraMovement::Zoom);
 
-    // We want to zoom in when we use mouse wheel up
+    // We want to zoom in when we use mouse wheel up,
     // so we increase the scale proportionally
-    // Note that the projections scale should always be positive (or our images will flip)
+    // Note that the projection scale should always be positive (or our images will flip)
```
Comment (alice-i-cecile) [inline examples/mouse_wheel.rs:58]:
```suggestion
    // Note that the projection's scale should always be positive (or our images will flip)

```

File: src/clashing_inputs.rs [hunk 8]
```diff
@@ -16,7 +16,7 @@ use std::cmp::Ordering;
 /// How should clashing inputs by handled by an [`InputMap`]?
 ///
 /// Inputs "clash" if and only if one [`UserInput`] is a strict subset of the other.
-/// By example:
+/// For example:
```
Comment (alice-i-cecile) [inline src/clashing_inputs.rs:19]: This is a slightly different meaning: "by example" implies "let me prove this", while "for example" means "let me teach you".

File: src/input_mocking.rs [hunk 9]
```diff
@@ -246,9 +246,9 @@ impl MockInput for MutableInputStreams<'_> {
     }
 
     fn reset_inputs(&mut self) {
-        // WARNING: this *must* be updated when MutableInputStreams's fields change
+        // WARNING: this *must* be updated when MutableInputStreams' field changed
```
Comment (alice-i-cecile) [inline src/input_mocking.rs:249]:
```suggestion
        // WARNING: this *must* be updated when MutableInputStreams''s fields change
```
This suggestion was wrong.

File: src/user_input.rs [hunk 10]
```diff
@@ -34,17 +34,18 @@ pub enum UserInput {
 }
 
 impl UserInput {
-    /// Creates a [`UserInput::Chord`] from a [`Modifier`] and an `input` that can be converted into an [`InputKind`]
+    /// Creates a [`UserInput::Chord`] from a [`Modifier`] and an `input` convertible into an [`InputKind`]
```
Comment (alice-i-cecile) [inline src/user_input.rs:37]:
```suggestion
    /// Creates a [`UserInput::Chord`] from a [`Modifier`] and an `input` which can be converted into an [`InputKind`]
```

**Review by alice-i-cecile — APPROVED** [review]
Generally mildly positive! Good job grammar checker tool :) And thanks for submitting these.

**Comment by Shute052** [comment]
added an extra apostrophe to MutableInputStreams

**Pull request merged**